conf/are.conf.part

```diff
@@ -4,7 +4,7 @@
 
 source src_ch_are_landschaftstypen : def_queryable_features
 {
-    sql_db = are
+    sql_db = are_dev
```

Comment:
Hmm. But before, when on production, we had `pgcluster.prod.bgdi.ch -> are`; now we would have `pg.bgdi.ch -> are_dev`, not?